pkg/cmd/run.go

```diff
@@ -685,6 +706,8 @@ func init() {
 	runCmd.Flags().String("mount-format", "json", fmt.Sprintf("file format to use. if not specified, will be auto-detected from mount name. one of %v", models.SecretsMountFormats))
 	runCmd.Flags().String("mount-template", "", "template file to use. secrets will be rendered into this template before mount. see 'doppler secrets substitute' for more info.")
 	runCmd.Flags().Int("mount-max-reads", 0, "maximum number of times the mounted secrets file can be read (0 for unlimited)")
+	runCmd.Flags().StringSliceVarP(&secretsToInclude, "only-secrets", "", []string{}, "only include the specified secrets")
+	runCmd.Flags().Bool("no-exit-on-missing-included-secrets", false, "do not exit on missing secrets via --only-secrets")
```

Comment:
Should we also rename this to `no-exit-on-missing-only-secrets`?

Reply:
I had thought of simply having `--no-exit` here: https://github.com/DopplerHQ/cli/pull/341#issuecomment-1303916724
`--no-exit-on-missing-only-secrets` works too, it feels long, would `--no-exit` be too general?

Reply:
`--no-exit` would definitely be too general, as it could apply to any number of options. I think we go with the longer name for now as I foresee users mostly using this inside a script.

Reply:
Awesome! Thank you @Piccirello